### ISSUE TITLE
Can't get tests running with Angular 13.0.3

### ISSUE THREAD

**Issue by Alfagun74** [dbfannin/ngx-logger#280]
Hi i just updated my Project to Angular 13.0.3 and got a  failing test using the LoggerTestingModule like this:

```typescript
import { LoggerTestingModule } from 'ngx-logger/testing';

[...]

TestBed.configureTestingModule({
      imports: [
        LoggerTestingModule,
      ],
      providers: [ ],
      declarations: [AppComponent],
    }).compileComponents();

    fixture = TestBed.createComponent(AppComponent);

    component = fixture.componentInstance;

[...]
```

The Error when running the tests is:
```typescript
AppComponent › ngOnInit › someTest

    Unexpected value 'LoggerTestingModule' imported by the module 'DynamicTestModule'. Please add an @NgModule annotation.

      53 |     }).compileComponents();
      54 |
    > 55 |     fixture = TestBed.createComponent(AppComponent);
         |                       ^
      56 |     component = fixture.componentInstance;
      57 |   });
```

Am i doing something wrong or does this library need an update for Angular 13 support?

**Comment by bmtheo** [comment]
Just published 5.0.4 that builds the lib with angular v13, can you check if this fixes the issue ?

**Comment by Alfagun74** [comment]
> Just published 5.0.4 that builds the lib with angular v13, can you check if this fixes the issue ?
Thanks man. I'll write here tomorrow @ work when i checked it

**Comment by Alfagun74** [comment]
@bmtheo works great now, good job!
